### ISSUE TITLE
Error running query: datetime.time(13, 52, 27) is not JSON serializable

### ISSUE THREAD

**Issue by SomeKittens** [EverythingMe/redash#463]
My table schema:

```sql
CREATE TABLE F_entrances (
  id SERIAL PRIMARY KEY,
  timeOfEntrance time,
  customerId int REFERENCES D_customers
);
```

(and yes, I committed the horrible sin of camel_case vs underScore.  I'll be fixing that soonish)

The query

```sql
SELECT
timeofentrance
FROM F_entrances
```

Gives me the error `Error running query: datetime.time(13, 52, 27) is not JSON serializable`.  I worked around it with `to_char` but this seems to be a problem at the [Python layer](http://stackoverflow.com/a/11875813/1216976).

**Comment by arikfr** [comment]
It's definitely is. What datasource type are you using?

**Comment by arikfr** [comment]
Ok, I believe I know where the problem is. Expect a fix tomorrow.

**Comment by SomeKittens** [comment]
I'm using postgres here.

**Comment by arikfr** [comment]
Fixed this in #464 and it's part of the [0.6.1-RC release](https://github.com/EverythingMe/redash/releases/tag/v0.6.1-rc). See details on how to upgrade [here](http://redash.io/deployment/upgrade.html).

There is still some room for improvement in the UI  (time columns treated as plain strings), but at least it will return results now.
